scripts/dev/import-conversations-unified-old.py

```
@@ -0,0 +1,887 @@
+from unified_state_manager import UnifiedStateManager
+
+from qdrant_client import QdrantClient
+from qdrant_client.models import PointStruct, Distance, VectorParams
+
+# Import normalize_project_name from shared module
+# Add parent directory to path to import shared module
+sys.path.insert(0, str(Path(__file__).parent.parent))
+try:
+    from shared.normalization import normalize_project_name
+except ImportError as e:
+    logging.error(f"Failed to import normalize_project_name from shared module: {e}")
```

Ruff (0.13.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)